apps/banking-mobile/.env.example

```
@@ -0,0 +1,12 @@
+# Banking Mobile — SANDBOX environment template
+# Copy to .env.local — NEVER commit .env.local
+# Expo reads EXPO_PUBLIC_* vars into the client bundle at build time.
+
+# LiteLLM proxy (banxe-general, :4000)
+EXPO_PUBLIC_BANKING_BACKEND_URL=http://localhost:4000/v1/chat/completions
+EXPO_PUBLIC_BANKING_MODEL=banxe-general
+
+# Keycloak sandbox (:8180) — sandbox realm only, no real customers
+EXPO_PUBLIC_KEYCLOAK_URL=http://localhost:8180
+EXPO_PUBLIC_KEYCLOAK_REALM=banxe-sandbox
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_KEYCLOAK_REALM key should go before the EXPO_PUBLIC_KEYCLOAK_URL key

(UnorderedKey)